contribute/en/Style-and-Tone.md

```diff
@@ -83,17 +83,19 @@ Refer to UI labels in inverted commas (eg 'click on the “mute” button')
 
 Jamulus is “Free and Open Source (FOSS)” (not “free software” or “open source”)
 
-“Channel” The audio signal as part of a mix. “Mute a channel”, “Maximum number of channels”, “Group channels together” (not “Mute a person” because one person might be using multiple channels).
+“Channel” The audio signal as part of a mix. “Mute a channel”, “Maximum number of channels”, “Group channels together”.
 
-“Fader” The UI that controls a channel. “Each fader has a mute button”, “The person’s fader” ,“Group faders together” (not “The person’s channel” or “Mute a Fader”, not “Slider” or “Volume control”)
+"Fader" - the UI control that adjusts the channel volume heard in the mix. Not "volume slider", "volume control" or other term (adding the context of the mixer, if needed).
```

Comment:
Why change to flat quotes here? There seems to be a great affection for curly quotes in these pages, though I'm not aware of a formal style preference.

Reply:
We may as well have one or the other, so I've changed them all to flat now.